tripy/tripy/frontend/module/parameter.py

```diff
@@ -31,7 +31,9 @@ class Parameter(Tensor):
     """
 
     @frontend_utils.convert_inputs_to_tensors()
-    def __init__(self, tensor: "tripy.Tensor") -> None:
+    def __init__(
+        self, tensor: Union["tripy.Tensor", "np.ndarray", "cp.ndarray", "torch.Tensor", "jnp.ndarray"]
```

Comment:
Maybe we can make the type annotation `Any`? It's hard to extensively list all types that support the DLPack interface

Reply:
I copied it over from the `Tensor` constructor but `Any` would be justified here, yes.

Reply:
Replaced with `Any`.